setup.py

```diff
@@ -389,6 +389,7 @@ def make_relative_rpath(path):
         # On Cloud TPU VM install with:
         # $ sudo pip3 install torch_xla[tpuvm] -f https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch_xla-1.11-cp38-cp38-linux_x86_64.whl
         'tpuvm': [f'libtpu-nightly @ {_libtpu_storage_path}'],
+        'test': ["expecttest"],
```

Comment:
according to https://github.com/GoogleCloudPlatform/ml-testing-accelerators/pull/787#pullrequestreview-1262207965